Sources/Swift/Integrations/SessionReplay/SentryReplayOptions.swift

```diff
@@ -146,6 +147,11 @@ public class SentryReplayOptions: NSObject, SentryRedactOptions {
      */
     let maximumDuration = TimeInterval(3_600)
 
+    /**
+     * Used by hybrid SDKs to be able to configure SDK info for Session Replay
+     */
+    var sdkInfo: [String: Any]?
```

Comment:
`m`: We should set the intial value to the values of the SentryMeta.

Reply:
`m`: We should use SentrySDKInfo here, convert it to Swift, and make it public. That's also how Android does it. Unfortunately, that's not trivial.

Reply:
IMO, an unnecessary extra work. This option its how hybrid SDKs override the information for replay events. If not present the integration will use the default value from SentryMeta.

Reply:
SDKInfo will be public. I really would like to avoid adding this as a dict, because then we need a major to get this right. If we can keep SDKInfo internal, I would be OK with this workaround for now.

Reply:
Oh now, sdkInfo is not public. It needs the public modifier to be public to the end user. Unless im missing something.

Reply:
OK, then, I'm OK with the dict approach.